app/liquid/liquid_renderer.rb

```diff
@@ -44,6 +50,39 @@ def images
 
   private
 
+  def member_hash
+    return nil if @member.blank?
+    values = @member.attributes.symbolize_keys
+    values[:welcome_name] = [values[:first_name], values[:last_name]].join(' ')
+    values[:welcome_name] = values[:email] if values[:welcome_name].blank?
+    values
```

Comment:
I want this to use Eric's `name` methods on Member, but I'm worried if I merge dev into this it'll make a headache merging into `cache_invalidation`. What if I fix this after we merge into `cache_invalidation`?